app/locales/ru.json

```diff
@@ -0,0 +1,110 @@
+    "no": "нет",
+    "pausedUntil": "\nПауза - {{timeLeft}} до возобновления работы\nпо обычному графику",
+    "pausedIndefinitely": "Пауза на неопределённый срок",
+    "timeToNext": "\n{{timeLeft}} до {{breakType}}",
+    "microbreak": "микроперерыва",
+    "break": "перерыва",
+    "nextBreakFollowing": "\nСледующий перерыв через {{count}} микроперерыв",
+    "nextBreakFollowing_plural": "\nСледующий перерыв через {{count}} микроперерывов"
+  },
+  "break": {
+    "keepGoing": "...не, я просто продолжу",
+    "title": "Перерыв!"
+  },
+  "microbreak": {
+    "keepGoing": "...не, я просто продолжу",
+    "title": "Перерыв!"
+  },
+  "about": {
+    "stretchly": "stretchly",
+    "breakTimeReminderApp": "программа для напоминания о перерывах",
+    "toLearnMoreVisit": "Если хотите узнать больше о возможностях stretchly или о том, как помочь приложению, посетите",
+    "homepage": "домашнюю страницу",
+    "latestVersion": "последняя версия:",
+    "checking": "проверяем..."
+  },
+  "settings": {
+    "settings": "Настройки",
+    "microbreaks": "микроперерывы",
+    "secondsEveryBreak": "секундный перерыв каждые",
+    "minutes": "минут",
+    "dontLetMeSkipMicrobreaks": "отключить возможность пропуска микроперерывов",
+    "breaks": "перерывы",
+    "minutesBreakAfter": "минутный перерыв после",
+    "microbreaks2": "микроперерывов",
+    "dontLetMeSkipBreaks": "отключить возможность пропуска перерывов",
+    "(every": "(каждые",
+    "minutes)": "минут)",
+    "microbreaksEnabled": "включить микроперерывы",
+    "breaksEnabled": "включить перерывы",
+    "resetToDefaults": "настройки по умолчанию",
+    "cantDisableBoth": "Невозможно отключить оба типа перерывов."
+  },
+  "settings2": {
+    "settings": "Настройки",
+    "audioScheme": "аудио",
+    "crystalGlass": "хрусталь",
+    "windChime": "ветерок",
+    "ticToc": "тик так",
+    "silence": "тишина",
+    "colorScheme": "цветовая палитра",
+    "greenClouds": "зелёные <br/> облака",
+    "autumnBeBlessed": "счастливая осень",
+    "graphiteCrystal": "кристаллы <br/>графита",
+    "coffeeKisses": "кофейный <br/> поцелуй",
+    "morningSwim": "утренний <br/> заплыв",
+    "resetToDefaults": "настройки по умолчанию"
+  },
+  "settings3": {
+    "settings": "Настройки",
+    "miscellaneous": "прочие",
+    "breaksAreFullscreen": "полноэкранные перерывы",
+    "showBreakIdeas": "показывать идеи для перерывов",
+    "notifyBeforeBreakStarts": "уведомлять перед началом перерыва",
+    "notifyBeforeMicrobreakStarts": "уведомлять перед началом микроперерыва",
+    "monitorIdleTime": "считать бездействие за естественный перерыв",
+    "showOnAllScreens": "отображать на всех экранах",
+    "language": "язык",
+    "resetToDefaults": "настройки по умолчанию"
+  },
+  "utils": {
+    "secondsLeft": "{{count}} секунда осталась",
+    "secondsLeft_plural": "{{count}} секунд осталось",
+    "minutesLeft": "{{count}} минута осталась",
+    "minutesLeft_plural": "{{count}} минут осталось",
```

Comment:
With plurals, you should have 3 possibilities for Russian language: 
key_0, key_1, key_2, eg: minutesLeft_0, minutesLeft_1, minutesLeft_2

You can read more here: 
https://www.i18next.com/plurals.html#how-to-find-the-correct-plural-suffix

Let me know if you need help with this